### PR TITLE
include speakingurl

### PR DESCRIPTION
**Pull request by herrmannplatz**
remove speakingurl from webapp-module-external and make it a dependency in webapp. Created a PR on the speakingurl repo with the Omega fix we used to have in our repo. I also upgraded from 4.0.0 to 10.0.0. 

see https://github.com/pid/speakingurl/pull/86

### CODE REVIEW

File: test/unit_tests/util/UtilSpec.coffee [hunk 1]
```diff
@@ -916,3 +916,19 @@ describe 'z.util.add_http', ->
   it 'does not add https if present', ->
     url = 'https://wire.com/'
     expect(z.util.add_http url).toBe 'https://wire.com/'
+
+describe 'z.util.compare_names', ->
+  it 'René equals Rene', ->
+    expect(z.util.compare_names('René', 'Rene')).toBeTruthy()
+
+  it 'Παναγιὡτα equals Panagiota', ->
+    expect(z.util.compare_names('Παναγιὡτα', 'Panagiota')).toBeTruthy()
```
Comment (lipis) [inline test/unit_tests/util/UtilSpec.coffee]: Change it to: `ώ` instead of `ὡ` (in the description as well)

Comment (herrmannplatz) [reply test/unit_tests/util/UtilSpec.coffee]: done

**Comment by bennycode** [comment]
Still failing because of `Παναγιώτα`?

**Comment by herrmannplatz** [comment]
Yes, but it should work now. Forgot to bundle speakingurl.

**Pull request merged**